### DISCUSSION TITLE
What is needed in order to run the Samples solution?

### DISCUSSION THREAD

**Discussion by davidpetric** [xamarin/XamarinCommunityToolkit#1027] [Q&A] [answered]
Hi,

I am trying to run the Samples solution and I am receiving a lot of errors:
![image](https://user-images.githubusercontent.com/13138387/110103156-4b894180-7dae-11eb-864d-e863d2e79e66.png)

### Repo steps:
- Downloaded the source code from the main branch:
- I installed the Developer Pack recommended by Visual Studio when opening the solution
- Restored NuGet packages
- Clean && Rebuild.

Maybe I need to install something else ? What I am missing ?

Thanks in advance!

**Comment by pictos** [comment, marked as answer]
@davidpetric please make sure that you have the UWP SDKs installed on your machine, I believe that you need both the `17763` and `19041` versions. Also, the logs in the output window can be more helpful.

**Comment by davidpetric** [comment]
Hi @pictos,

First of all thank you for your time and the help provided. 

I solved the issue by installing the UWP SDK's you mentioned.
